crates/frost/src/mocks.rs

```diff
@@ -14,6 +14,52 @@ pub struct MockSignerUserStorage {
     user_states: Arc<Mutex<BTreeMap<String, SignerUserState>>>,
 }
 
+pub struct MockSignerSessionStorage {
+    session_state: Arc<Mutex<BTreeMap<(String, String), SignerSessionState>>>,
```

Comment:
maybe use just tuple with uuid already
```suggestion
    session_state: Arc<Mutex<BTreeMap<(Uuid, String), SignerSessionState>>>,
```

(you don't have to change it [just as a comment], because logic is wrapped around that user can have any id, whether it is "1" or some long uuid)